crontainer/task.go

```diff
@@ -43,13 +43,3 @@ func (t *Task) Run() {
 func (t *Task) log(text interface{}) {
 	log.Printf("[%15s] %s", t.Name, text)
```

Comment:
Could you shorten this to 8 characters? That would be the first character block before the hyphen. For the output that should be enough.
```go
t.Name[:8]
```